### ISSUE TITLE
Webpack issue with simple-peer-light

### ISSUE THREAD

**Issue by rogersanick** [dmotz/trystero#68]
Hi there, I have an on-going webpack issue with simple-peer-light. To fix this I've had to clone the repo and publish my own NPM package with the dependency swapped to just good ole' `simple-peer`.

The issue is as follow, with Trystero latest installed I get this issue:
```
/Users/nicholasrogers/Downloads/Coding/flourish_and_decay/node_modules/simple-peer-light/index.js:1171
export default Peer
^^^^^^

SyntaxError: Unexpected token 'export'
    at internalCompileFunction (node:internal/vm:128:18)
    at wrapSafe (node:internal/modules/cjs/loader:1279:20)
    at Module._compile (node:internal/modules/cjs/loader:1331:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
    at Module.load (node:internal/modules/cjs/loader:1205:32)
    at Module._load (node:internal/modules/cjs/loader:1021:12)
    at cjsLoader (node:internal/modules/esm/translators:366:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:315:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24) {
  page: '/'
}
 ⨯ /Users/nicholasrogers/Downloads/Coding/flourish_and_decay/node_modules/simple-peer-light/index.js:1171
export default Peer
^^^^^^

SyntaxError: Unexpected token 'export'
```

For additional context these are my dependencies:

```
{
  "name": "flourish-and-decay-next-v2",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "format": "prettier --check --ignore-path .gitignore .",
    "format:fix": "prettier --write --ignore-path .gitignore .",
    "typecheck": "tsc -b",
    "update:check": "npx taze"
  },
  "dependencies": {
    "@dnd-kit/sortable": "^8.0.0",
    "@funtech-inc/use-shader-fx": "^1.1.5",
    "@react-three/drei": "9.105.1",
    "@react-three/fiber": "8.16.1",
    "@react-three/postprocessing": "^2.15.12",
    "@types/three": "^0.160.0",
    "@vercel/analytics": "^1.1.1",
    "dat.gui": "^0.7.9",
    "eslint-config-prettier": "^9.0.0",
    "ethers": "^6.8.0",
    "fs-extra": "^11.2.0",
    "geist": "^1.1.0",
    "gsap": "^3.12.2",
    "lodash": "^4.17.21",
    "next": "13.5.6",
    "openai": "^4.28.4",
    "postprocessing": "^6.34.1",
    "prettier": "^3.1.0",
    "prettier-plugin-tailwindcss": "^0.5.7",
    "r3f-perf": "^7.1.2",
    "react": "^18.2.0",
    "react-beautiful-dnd": "^13.1.1",
    "react-dom": "^18.2.0",
    "react-icons": "^4.11.0",
    "react-slick": "^0.29.0",
    "react-swipeable": "^7.0.1",
    "react-transition-group": "^4.4.5",
    "tailwindcss-animated": "^1.0.1",
    "three": "^0.160.0",
    "trystero": "^0.18.0",
    "uuid": "^9.0.1",
    "zustand": "^4.4.7"
  },
  "devDependencies": {
    "@types/fs-extra": "^11.0.4",
    "@types/lodash": "^4.14.200",
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-beautiful-dnd": "^13.1.8",
    "@types/react-dom": "^18",
    "@types/react-slick": "^0.23.12",
    "@types/react-transition-group": "^4.4.8",
    "@types/uuid": "^9.0.6",
    "@typescript-eslint/eslint-plugin": "^7.0.1",
    "autoprefixer": "^10",
    "eslint": "^8",
    "eslint-config-next": "13.5.6",
    "postcss": "^8",
    "tailwindcss": "^3",
    "typescript": "^5"
  }
}
```

**Comment by dmotz** [comment]
The new version of Trystero fixes this (it no longer uses simple-peer or simple-peer-light).

If you're up for it, you can beta test it in your project by doing `npm i trystero@next`.

Please let me know what issues you encounter.

**Comment by rogersanick** [comment]
@dmotz  - thank you very much. I'll report back with any issues. Appreciate all you've built and continue to add to this project. My project wouldn't be possible without it!

**Comment by jeremyckahn** [comment]
> My project wouldn't be possible without it!

@rogersanick if you can, please consider adding your project to https://github.com/jeremyckahn/awesome-trystero. I’d like to showcase what people are building with this awesome library!

**Comment by rogersanick** [comment]
@jeremyckahn - will do! Thanks for sending me the link.

**Comment by rogersanick** [comment]
@dmotz - I tried trystero@next with a small group of play testers. All using chrome with mixes OS. No build issue, which is awesome, but connection failed to establish between clients. I didn't see any issues in the console or any errors thrown.

Fwiw - two of the play testers were in Montreal and one was in Vancouver. I reverted to the v0.18.0 release using the Nostr strategy and it worked great.
